packages/diagrams/backend/sirius-components-diagrams/src/main/java/org/eclipse/sirius/components/diagrams/InsideLabel.java

```diff
@@ -25,19 +25,13 @@
 @Immutable
 public final class InsideLabel {
     private String id;
-
     private String type;
```

Comment:
The `InsideLabel.type` field is now only relevant for Sprotty, right?

Reply:
Yes it is, it may be removed later